link/perf_event.go

```diff
@@ -432,3 +432,17 @@ func isValidTraceID(s string) bool {
 
 	return true
 }
+
+// getTracefsPath will return a correct path to the tracefs mount point.
+// Since kernel 4.1 tracefs should be mounted by default at /sys/kernel/tracing,
+// but may be also be available at /sys/kernel/debug/tracing if debugfs is mounted.
+// The available tracefs paths will depends on distribution choices.
+func getTracefsPath() string {
+	if tracefsPath == "" {
```

Comment:
This creates a race condition, please remove tracefsPath and use sync.Once. Example from the codebase: https://github.com/cilium/ebpf/blob/63a3caefca29ce4a580a033044c71b57eb034092/internal/cpu.go#L10-L24

Reply:
Oh right .. Thanks for the catch and suggestion, this should be ok now.